packages/react-components/src/components/chart/utils/getXAxis.ts

```diff
@@ -19,6 +19,6 @@ export function getXAxis(axis?: ChartAxisOptions): XAXisOption {
     },
     splitNumber: 6,
     min: 0,
-    max: Date.now(),
+    max: viewportInMs.isDurationViewport ? viewportInMs.end : undefined,
```

Comment:
does this break gesturing? specifically scrolling to the left and right on the charts